docs/manifest.json

```diff
@@ -2427,6 +2427,74 @@
                 ]
               ]
             },
+            {
+              "title": "Vue",
+              "collapse": true,
+              "icon": "vue",
+              "items": [
+                [
+                  { "title": "Overview", "href": "/docs/references/vue/overview" },
+                  {
+                    "title": "Guides",
+                    "items": [
+                      [
+                        {
+                          "title": "Migrating from community SDK",
+                          "href": "/docs/references/vue/migrating-from-vue-community-sdk"
+                        }
+                      ]
+                    ]
+                  },
+                  {
+                    "title": "Client-side Helpers",
+                    "items": [
+                      [
+                        {
+                          "title": "`useUser()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-user"
+                        },
+                        {
+                          "title": "`useClerk()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-clerk"
+                        },
+                        {
+                          "title": "`useAuth()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-auth"
+                        },
+                        {
+                          "title": "`useSignIn()`",
+                          "wrap": false,
+                          "href": "/docs/references/vue/use-sign-in"
+                        },
```

Comment:
PR for the ff. pages [here](https://github.com/clerk/clerk-docs/pull/1711)

Reply:
I love how you broke these PRs out. Let's also add the relevant manifest links to each PR so we can preview while reviewing.

Reply:
Thanks! Are you talking about putting the client-side helper manifest links to this PR's description too?

Reply:
To each PR that has new links in the manifest.

For example, I'm looking at `useAuth()` for this PR: https://github.com/clerk/clerk-docs/pull/1710. This way, I can view the helper functions locally. :)

Reply:
Ah gotcha!